extensions/jni/jvm/JniReferenceObjects.h

```diff
@@ -128,8 +128,8 @@ class JniByteInputStream {
     return read;
   }
 
-  int64_t read(uint8_t &arr) {
-    return stream_->read(arr);
+  int64_t read(uint8_t &arr) const {
```

Comment:
also, nitpicking and old code, but it's weird to call a reference-to-byte parameter "arr"